scripts/README.md

```diff
@@ -14,11 +14,14 @@
    `go run ./scripts/token/associate/associate.go --privateKey=/your private key/ --accountID=/your account id/ --network=/previewnet|testnet|mainnet/ --tokenID=/The Token id from the output of the previous step/`
 
 ## Update bridge config
-1. Run prepare-update-config.go with executorAccountID, topicID and configPath to create frozen transaction as bytes which to send for signing to all members
+1. Run create-topic.go (only once) with supplyKeys, executorAccountID, executorPrivateKey, keyThreshold, network to create the topic with all members public keys and the key threshold.
+   `go run ./scripts/common/create-topic/create-topic.go --supplyKeys /members public keys/ --executorAccountID /executor account id/ --executorAccountID /executor private key/ --keyThreshold /key threshold (default: 1)/ --network /previewnet|testnet|mainnet/`
+
+2. Run prepare-update-config.go with executorAccountID, topicID and configPath to create frozen transaction as bytes which to send for signing to all members
    `go run ./scripts/bridge/update-config/prepare-update-config.go --executorAccountID /executor account id/ --topicID /topic id/ --network /previewnet|testnet|mainnet/ --configPath /full or relative path to bridge.yml/ --validStartMinutes /start minutes (default 2)/ --nodeAccountID /node account id which process the transaction (default: "0.0.3")/`
 
-2. Every member of the topic needs to run sign on the generated bytes by running sign-transaction.go with privateKeys and transaction (the bytes)
+3. Every member of the topic needs to run sign on the generated bytes by running sign-transaction.go with privateKeys and transaction (the bytes)
    `go run ./scripts/common/sign/sign-transaction.go --privateKeys /member private key (or all members private keys separated by comma/ --transaction /transaction bytes/`
```

Comment:
`go run ./scripts/common/sign/sign-transaction.go --privateKeys /private keys, which need to sign the transaction, separated by comma/ --transaction /transaction bytes/`